app/models/style_guide/coffee_script.rb

```
@@ -26,7 +26,11 @@ def lint(content)
 
     def content_for_file(file)
       if erb? file
-        ERB.new(file.content).result
+        begin
+          ERB.new(file.content).result
+        rescue
```

My thought is to just rescue `NoMethodError` and see if that is good enough. The thought here is, if there are other errors this code is raising, then we may want to fix them instead of silently ignoring them.